### ISSUE TITLE
Textures aren't visible after I imported new map.

### ISSUE THREAD

**Issue by errkedukke** [Pottus/Texture-Studio#104]
So yeah, Textures aren't seen do you guys know why it is? I'm using 

SetDynamicObjectMaterial method

**Comment by errkedukke** [comment]
Yes format is the same that is in exported maps.

**Comment by Crayder** [comment]
- What version of TS are you using?
- Could you provide the code that you are importing?
- Have you tried with other map code?
- Have you done any other troubleshooting aside from checking to see if the format matched up? You're not giving me much to go off of here...
- Do you lines possibly have no spaces (this definitely matters for material importing)? (If this is the issue then you may also have a similar issue with RemoveBuilding lines.)

**Comment by Crayder** [comment]
@errkedukke Did you figure it out?

**Comment by errkedukke** [comment]
Nah, There are some kinda bugs with texts - they cant be hidden plus when
you import map textures are set but after you save the map in .db file they
are still missing. This happens when I import the map. So yeah I gave up

**Comment by Crayder** [comment]
You gave up and closed the issue? That's not how troubleshooting works...

So it only occurs when you have imported the map? Does it happen with new maps?

**Comment by Fantasztikus21** [comment]
Same problem. After i finish a map and i want import to the another map. 
The first import is text not loaded. 
I closed and re-open the map then all textures is gone. 
I tryed another map and i got same problem every time.

- I used texture studio 1.9b and 0.3.7-R2 samp server.

some pictures of the bug:

First import text not imported.
#1 [Image ](https://imgur.com/yVy5MzS)

Second i closed and open the map textures are gone.
#2 [Image](https://imgur.com/pKiprnq)

I got another bug. The server is logged the "/thelp missing command: mtsearch".
# [Server log](https://pastebin.com/8hQF6BB9)
